plugins/src/lib.rs

```diff
@@ -345,6 +358,7 @@ where
             hooks: self.hooks.keys().map(|s| s.clone()).collect(),
             rpcmethods,
             notifications: self.notifications.clone(),
+            featurebits: self.featurebits.clone().to_message(),
```

Comment:
Is this `clone` really needed? Can we not call the `to_message()` method that returns a serializable struct (same struct as the "builder" but serde annotated) without cloning the builder struct?

I guess the "builder" is just for separation of concerns, as it contains the exact same fields as the message type?

Reply:
I see your point here.

The `FeatureBitBuilder` is indeed for separation of concerns and might be overkill here.

The problem here is not in the conversion of `to_message()`. The core problem is that the `Builder` must outlive the `GetManifestResponse`.

We could add a lifetime to `GetManifestResponse` and borrow the `FeatureBits` instead.
However, I don't think it's worth the engineering-time to save a few allocations at plugin-start.